### ISSUE TITLE
Change Language

### ISSUE THREAD

**Issue by brandynn** [vincentscode/ExtendedKawaii-DiscordBot#10]
Hey! it's awesome that you have made a replacement for kawaii bot, but it's in german and that's sort of an issue haha- could we have a setting to change it back to english? thanks :)

**Comment by vincentscode** [comment]
Hey, absolutely!
Originally this was not meant to replace kawaii bot but just extend it which is why something like that does not exist.
Would you mind adding some translations if i were to provide some sort of localization file?

**Comment by brandynn** [comment]
yeah, thats what i meant, extending it :) which is great! and i'm really happy you're doing it :DD - I can try help translate, do you not have the original english file for the bot? I can ask the creator.

**Comment by vincentscode** [comment]
Well the original bot is open source so the files should be available to me.
But my bot has different / more commands and stuff so a lot will have to be redone.
I will look into how hard it is to make it localizable and let you know!

**Comment by brandynn** [comment]
oh okay :) sweet definitely will try and help
